test/runtests.jl

```diff
@@ -1,12 +1,59 @@
 using Test
-using LinearAlgebra, Statistics, TopoPlots, CairoMakie
+using LinearAlgebra, Statistics, TopoPlots, CairoMakie, FileIO
+using PyCall
+try
+    PyCall.pyimport("matplotlib")
+catch e
+    # I tried adding Conda for PyPlot, which then installs matplotlib automatically.
+    # It looks like this messed with mne, so that then needed manual installation...
+    # Now, Conda started making problems (in a fresh CI env?!) https://github.com/MakieOrg/TopoPlots.jl/pull/20#issuecomment-1224822002
+    # So, lets go back to install matplotlib manually, and let mne install automatically!
+    run(PyCall.python_cmd(`-m pip install matplotlib`))
+end
+import PyPlot, PyMNE
+PyPlot.pygui(false)
 
 include("percy.jl")
 
 data, positions = TopoPlots.example_data()
 
+function mne_topoplot(fig, data, positions)
+    circle = TopoPlots.enclosing_geometry(Circle, positions)
+    # Seems like the only way to plot positions with matching head is to norm the positions..
+    # Anything else leads to anarchy!
```

Comment:
how does this relate to #18?

Reply:
Good question... I don't think it does, but I don't know MNE well enough, to know what's going wrong on there side.
I'll take a look at #18 after this PR!